themes/default/content/blog/examining-recommended-practices-code-placement-and-stacks/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Examining Recommended Practices: Code Placement and Stacks"
```

Comment:
Just thinking in terms of SEO and anticipating user questions: Do you think it's more likely someone would use the words "code placement" than "code organization" (or similar) when looking for guidance on this subject? I'd probably look for "code organization" myself -- and a quick search through Community Slack suggests that's a phrase others seem to use pretty commonly as well. What do you think about changing this?

 ```suggestion
title: "Examining Recommended Practices: Code Organization and Stacks"
```

Reply:
Fixed in 3b48f0ca